js/index.js

```diff
@@ -2,41 +2,56 @@
 
 function updateSubtotal(product) {
   console.log('Calculating subtotal, yey!');
+  const price = product.querySelector('.price span').innerText;
+  const quantity = product.querySelector('.quantity input').value;
+  const subtotal = price * quantity;
+  product.querySelector('.subtotal').innerText = subtotal;
```

Comment:
In this case, you could target the span elements inside the subtotal class, in order to leave the dollar sign next to the subtotal value! Ex: $ <td class="subtotal"> $<span>40</span></td>. This way, you'd only change the value inside the span 😀

Reply:
ok, just changed, thanks!